docs/usage/configuration-templates.md

```diff
@@ -5,57 +5,66 @@ description: How to edit Renovate's config templates
 
-Most people can leave `branchPrefix` as `renovate/` however if you prefer to have no forward slashes then you might pick `renovate-` instead.
-Please note that the onboarding PR is fixed to use `renovate/configure` however.
+Most users will be happy with the default `branchPrefix` of `renovate/`, but you can change this if you don't like the default.
+Say you don't want the forward slashes, in that case you would use `renovate-` as your `branchPrefix`.
+The onboarding PR will always use `renovate/configure`.
 
 `additionalBranchPrefix` is optional and by default is empty for all JavaScript dependencies.
-We use `docker-` for all Docker updates, so you might see branches like `renovate/docker-ubuntu-16.x`.
+We use `docker-` for all Docker updates, branches will look like this: `renovate/docker-ubuntu-16.x`.
 
 `branchTopic` depends on the package manager and upgrade type, so you will see a lot of variety.
-It is also the one you might be most likely to want to change, but be careful and consider posting your config to https://github.com/renovateapp/config-help first.
+This is probably a setting you want to change yourself.
+Be careful, and consider posting your config to https://github.com/renovateapp/config-help first to get help from the Renovate team with your config.
 
 ## Commit Message
 
-Renovate is designed to have just a single commit per branch, for merging convenience.
-As such, that commitMessage should reflect the contents of the branch and usually be the same as the PR Title.
+Renovate will use one commit per branch, this makes it easy for you to merge.
+As such, the `commitMessage` reflects the contents of the branch and is usually the same as the PR title.
 
 `commitMessage` has a default value of `{{commitMessagePrefix}} {{commitMessageAction}} {{commitMessageTopic}} {{commitMessageExtra}} {{commitMessageSuffix}}`, with the intention that you only edit some of those subcomponents.
 
-`commitMessagePrefix` is usually not necessary to configure directly, and is used by Renovate if it needs to add a prefix due to Semantic Commit conventions.
-Do not touch it unless you know what you're doing.
+You usually don't need to edit `commitMessagePrefix`, this option is used by Renovate if it needs to add a prefix to conform to the Semantic Commit convention.
+Do not touch this unless you know what you're doing.
 
-`commitMessageAction` is usually just 1 word, e.g. 'Update', 'Pin', 'Refresh', etc.
-It's usually also not necessary to edit, although maybe you prefer 'Upgrade' instead of 'Update'?
+`commitMessageAction` is usually just one word, e.g. 'Update', 'Pin', 'Refresh', etc.
+You're probably fine leaving this setting alone, though you can change it.
+e.g. if you prefer that Renovate uses the term 'Upgrade' instead of 'Update'
+TODO: Explain how user can change `commitMessageAction` for 'Update' -> 'Upgrade'
```

Comment:
```suggestion
e.g. if you prefer that Renovate uses the term 'Upgrade' instead of 'Update' then you could configure `"commitMessageAction": "Upgrade"`.
```